jumpscale/packages/vdc_dashboard/chats/matic.py

```diff
@@ -39,13 +38,22 @@ def get_config(self):
             "env.sentry_enodeid": self.config.chart_config.extra_config.get("sentry_enodeid", ""),
         }
 
+    def get_config_string_safe(self):
+        return {
+            "env.heimdall_svcp": str(self.config.chart_config.extra_config["heimdall_svcp"]),
+            "env.bor_svcp": str(self.config.chart_config.extra_config["bor_svcp"]),
+            "ports.heimdall": str(self.config.chart_config.extra_config["heimdall_svcp"]),
+            "ports.bor": str(self.config.chart_config.extra_config["bor_svcp"]),
+        }
+
+    @chatflow_step(title="Node Configuration")
```

Comment:
the `chatflow_step` decorator shouldn't use with a private function so this line should move to `set_config` function line 123
```suggestion
```

Reply:
Moved as suggested